From 0ab7ea56b2558d30400a7462a05014e758c9c9c1 Mon Sep 17 00:00:00 2001
From: Matt Turner <mattst88@gmail.com>
Date: Wed, 10 Jan 2024 10:56:17 -0500
Subject: [PATCH] symbols-check: Add _GLOBAL_OFFSET_TABLE_

This is exported on hppa/parisc.

See also: https://gitlab.freedesktop.org/glvnd/libglvnd/-/merge_requests/291

Cc: mesa-stable
Bug: https://bugs.gentoo.org/908079
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/26978>
---
 bin/symbols-check.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bin/symbols-check.py b/bin/symbols-check.py
index 7daf603c5f9..2e014731728 100644
--- a/bin/symbols-check.py
+++ b/bin/symbols-check.py
@@ -7,6 +7,7 @@ import subprocess
 
 # This list contains symbols that _might_ be exported for some platforms
 PLATFORM_SYMBOLS = [
+    '_GLOBAL_OFFSET_TABLE_',
     '__bss_end__',
     '__bss_start__',
     '__bss_start',
-- 
2.41.0

